lib/ts/recipe/session/index.ts

```diff
@@ -53,6 +53,19 @@ export default class SessionAPIWrapper {
     static signOut(): Promise<void> {
         return Session.getInstanceOrThrow().signOut();
     }
+
+    static validateClaims(claimValidators: SessionClaimValidator<any>[]): Promise<ClaimValidationError | undefined> {
+        return Session.getInstanceOrThrow().validateClaims(claimValidators);
+    }
+
+    // have backwards compatibility to allow input as "signin" | "signup"
+    static redirectToAuth(input?: { redirectBack?: boolean }): Promise<void> {
+        if (input?.redirectBack !== true) {
+            return Session.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath();
+        } else {
+            return Session.getInstanceOrThrow().redirectToAuthWithRedirectToPath();
+        }
+    }
```

Comment:
@porcellus missed this?

Reply:
We've discussed removing recipe-specific auth components and simply redirecting users to websiteBasePath instead of using the recipe id of the current auth component.

Reply:
We do not need to expose this to the user from the index.ts file